requirements/runtime.txt

```diff
@@ -123,8 +116,6 @@ rich==13.9.4
     # via
     #   sigstore
     #   twine
-secretstorage==3.3.3
-    # via keyring
```

Comment:
NB: You might need to re-run `pip-compile` on a Linux container or similar, per https://github.com/pypa/gh-action-pypi-publish/pull/359#discussion_r2092154129

Reply:
Fixed!